src/families/filecoin/test-dataset.ts

```diff
@@ -0,0 +1,219 @@
+import { BigNumber } from "bignumber.js";
+import {
+  AmountRequired,
+  FeeNotLoaded,
+  InvalidAddress,
+  InvalidAddressBecauseDestinationIsAlsoSource,
+  NotEnoughBalance,
+} from "@ledgerhq/errors";
+
+import type { DatasetTest } from "../../types";
+import type { Transaction } from "./types";
+import { fromTransactionRaw } from "../filecoin/transaction";
+
+const SEED_IDENTIFIER = "f1zx43cf6qb6rd5e4okl7lexnjumxe5toqj6vtr3i";
+const ACCOUNT_1 = "t1d2xrzcslx7xlbbylc5c3d5lvandqw4iwl6epxba";
+
+const dataset: DatasetTest<Transaction> = {
+  implementations: ["js"],
+  currencies: {
+    filecoin: {
+      FIXME_ignoreAccountFields: [],
+      scanAccounts: [
+        {
+          name: "filecoin seed 1",
+          apdus: `
+          => 0600000000
+          <= 0000120500311000049000
+          => 06010000142c000080cd010080000000800000000000000000
+          <= 041e10b3a453db1e7324cd37e78820d7d150c13ba3bf784be204c91afe495816a19a836e85cb89b0e4ff36d06a71a9ca02947de79e16e66dacc645e46dcdf7d9091501cdf9b117d00fa23e938e52feb25da9a32e4ecdd02966317a78343363663671623672643565346f6b6c376c65786e6a756d786535746f716a3676747233699000
+          `,
+        },
+      ],
+      accounts: [
+        {
```

Comment:
if you think "balance is sum of ops" can't be true, you can add here a

```
          FIXME_tests: [
            "balance is sum of ops",
          ],
```

like done at https://github.com/LedgerHQ/ledger-live-common/blob/9dbfe82ca0511b197d779924ec52c2efa36cb122/src/families/tezos/test-dataset.ts#L63

for CI to ignore this to test.